src/zope/app/apidoc/codemodule/directives.rst

```diff
@@ -48,9 +48,12 @@ registry tries to import the path. The hook was provided for security reasons,
 since uncontrolled importing of modules in a running application is considered
 a security hole.
 
-By default the flag is set to false:
+By default the flag is set to false (of course, this depends on the
+order in which tests are run and what ZCML has been configured or if
+this was manually changed, so we can't really rely on the default here):
```

Comment:
I don't remember exactly where it was, but there was a test that wasn't tearing its setting change down explicitly. It *may* have been the static export, and I *think* I fixed that. But I'm not exactly sure.